package.json

```
@@ -78,7 +78,7 @@
   "devDependencies": {
     "@babel/core": "^7.0.0",
     "@babel/preset-env": "^7.0.0",
-    "@jupyterlab/builder": "^3.1.0",
+    "@jupyterlab/builder": "^4.0.0-alpha.11",
```

```suggestion
    "@jupyterlab/builder": "^3.4.7",
```

Downgrading the version to `^3.4.7`, as the build output suggests, causes the build to pass (with warnings) for me locally.

---

Thanks! 👍